### ISSUE TITLE
The "APPLY" button only works half the time why??? also the chat has become dumber why? please anyone from team listen

### ISSUE THREAD

**Issue by fisforfaheem** [getcursor/cursor#2474]
I'm a paying customer with a team of 100+ users. The reliability of your product has been atrocious lately, and it's unusable for us. We're now considering switching to Windsurf. Why isn't your product working? Please address these basic issues ASAP, and please respond to my question.

**Comment by saurabhkumar8112** [comment]
yeah they have made it really bad with recent updates and the team doesn't seem to care. I am looking for alternatives as well
